Update the day of the biweekly meeting (#15)
kserve/community: README.md

## Community meeting

> We have public biweekly community meetings on Thu 9AM US/Pacific. Please [map that to your local time](https://www.google.com/search?q=0900+am+in+pst&hl=en).

Reviewer: ```suggestion
We have public biweekly community meetings on Thursdays at 9AM PST. Please [map that to your local time](https://www.google.com/search?q=0900+am+in+pst&hl=en).
```
